spec/mailers/reminder_spec.rb

```diff
@@ -17,7 +17,8 @@
   end
 
   it "should include link in body" do
-    expect(email).to have_body_text('http://ombulabs.letsfreckle.com')
+    ENV["FRECKLE_ACCOUNT_HOST"] = 'foobar'
```

Comment:
@schmierkov you shouldn't modify `ENV` like that in a scenario. It may cause problems in another scenario because it's a global constant. 

Please stub it properly: https://robots.thoughtbot.com/testing-and-environment-variables